tests/test_bigs.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 import json
 import os
+drr = '/Users/olawumiolasunkanmi/Library/CloudStorage/OneDrive-UniversityofNorthCarolinaatChapelHill/FALL2022/BACKUPS/ARAGORN/AnswerCoalesce'
```

Comment:
I don't understand what's going on here.  And it looks like the input_dir doesn't exist any more?

Reply:
My mistake. I was having directory issues initially, I was supposed to delete the line before the git push.